themes/bootstrap3/templates/myresearch/menu.phtml

```diff
@@ -1,116 +1,14 @@
 <?php
   $user = $this->auth()->isLoggedIn();
-  $patron = $user ? $this->auth()->getILSPatron() : false;
-  $capabilityParams = $patron ? ['patron' => $patron] : [];
-  $ilsOnline = ('ils-none' !== $this->ils()->getOfflineMode());
 ?>
+
```

Comment:
Is this blank line needed? We don't usually have whitespace between the initial PHP block and subsequent markup.

Reply:
blank line removed